karoo-gp.py

```diff
@@ -555,12 +534,30 @@ def fx_karoo_pause(model):
 
     return 1
 
+#++++++++++++++++++++++++++++++++++++++++++
+#   Load Data                             |
+#++++++++++++++++++++++++++++++++++++++++++
+karoo_dir = os.path.dirname(os.path.realpath(__file__))
+suffix = dict(r='REGRESS', c='CLASSIFY', m='MATCH', p='PLAY')[kernel]
+func_path = karoo_dir + f'/karoo_gp/files/operators_{suffix}.csv'
+filename = filename or karoo_dir + f'/karoo_gp/files/data_{suffix}.csv'
```

Comment:
This should use `pathlib`

Reply:
I'm still not 100% on the best way to do this. We should have a chat about file management in Karoo.

Reply:
The translation from `os.path` to `pathlib` is quite literal:
```py

karoo_dir = pathlib.Path(__file__).resolve().parent
suffix = dict(r='REGRESS', c='CLASSIFY', m='MATCH', p='PLAY')[kernel]
func_path = karoo_dir / 'karoo_gp' / 'files' / f'operators_{suffix}.csv'
filename = filename or karoo_dir / 'karoo_gp' / 'files' / f'data_{suffix}.csv'
```